src/core/Akka.Remote/Endpoint.cs

```diff
@@ -1350,14 +1286,14 @@ private void SendBufferedMessages()
                 {
                     _log.Debug("Drained buffer with maxWriteCount: {0}, fullBackoffCount: {1}," +
                                "smallBackoffCount: {2}, noBackoffCount: {3}," +
-                               "adaptiveBackoff: {4}", _maxWriteCount, _fullBackoffCount, _smallBackoffCount, _noBackoffCount, _adaptiveBackoffNanos / 100);
+                               "adaptiveBackoff: {4}", _maxWriteCount, _fullBackoffCount, _smallBackoffCount, _noBackoffCount, _adaptiveBackoffNanos / 1000);
```

Comment:
Fixed a small bug with the order of magnitude used for reporting adaptive backoff nanos.